packages/flutter/test/material/tabs_test.dart

```diff
@@ -3812,6 +3812,89 @@ void main() {
     final Tab firstTab = tester.widget(find.widgetWithIcon(Tab, Icons.check));
     expect(firstTab.height, 85);
   });
+
+  testWidgets('Test semantics of TabPageSelector', (WidgetTester tester) async {
```

Comment:
Can you add a test for one different locale?

Reply:
sure, i'm gonna add a new localization for page instead using tabLocalization and a test for a different locale soon, thanks for review @chunhtai

Reply:
oh you can use tabLocalization, I have verified with the original developer and they also agree it should be tab. you can ignore my previous comment

Reply:
@chunhtai Sorry for the delay, i noticed that to do a test for a different locale i have to use GlobalMaterialLocalizations and GlobalWidgetLocalizations as delegates who are localized in flutter_localizations package and i don't have that package in flutter test, so i tried to add the test in flutter_localizations package, but i dont have the SemanticsTester class right there, should i move this class to flutter_localizations package tests or that would be a bad approach?

Reply:
can you implement your own custom delegate and make sure it uses the string you put in there?

Reply:
Sure, i'll do that